pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -39,6 +39,12 @@ var (
 	sqlError = errs.Class("datastore-sql")
 )
 
+const (
+	MySQL      = "mysql"
+	PostgreSQL = "postgresql"
```

Comment:
This was previously "posgres".

Reply:
Yes, bad refactoring. Going to revert back to `postgres` 👍